VirtoCommerce.Storefront.Model/Cart/ShoppingCart.cs

```diff
@@ -53,7 +53,7 @@ public ShoppingCart(Currency currency, Language language)
         /// FluentValidation RuleSets allow you to group validation rules together which can be executed together as a group. You can set exists rule set name to evaluate default.
         /// <see cref="CartValidator"/>
         /// </summary>        
-        public string ValidationRuleSet { get; set; } = "default,strict"; //Achtung!!! It is very important not to use spaces for fluent validation  rule set string, all values after the space will be ignored!
+        public string ValidationRuleSet { get; set; } = "default,strict,shipment"; //Achtung!!! It is very important not to use spaces for fluent validation  rule set string, all values after the space will be ignored!
```

Comment:
Is it not necessary to change original storefront code. Revert and might be push PR to platform team.